### DISCUSSION TITLE
Single comments can no longer be sent continuously

### DISCUSSION THREAD

**Discussion by soranoba** [community/community#7574] [Pull Requests] [answered]
Until about 2~3 days ago, I was able to send SingleComments continuously, but now I can't.
I'm having a hard time reviewing

https://user-images.githubusercontent.com/4954289/141243284-3866dc2f-2fc9-4db9-a439-2340d02d66ed.mov

**Comment by Totktonada** [comment, marked as answer]
Duplicate of https://github.com/github/feedback/discussions/7481.

Workaround: refresh the page after each comment.

**Reply by soranoba** [reply]
Thank you for the information.
